app/models/person.rb

```diff
@@ -398,4 +408,11 @@ def other_person_with_same_guid
     diaspora_id = Person.where(guid: guid).where.not(diaspora_handle: diaspora_handle).pluck(:diaspora_handle).first
     errors.add(:base, "Person with same GUID already exists: #{diaspora_id}") if diaspora_id
   end
+
+  def self.diaspora_handle_from_blocked_pod?(diaspora_handle)
```

Comment:
Lint/IneffectiveAccessModifier: `private` (on line 395) does not make singleton methods private. Use `private_class_method` or `private` inside a `class << self` block instead.